update-ca-certificates

```diff
@@ -76,6 +76,12 @@ esac
 rm -f /etc/pki/trust/.updated
 
 mkdir -p "$statedir"
+if [ -z "$_CA_CERTIFICATES_LOCKED" ]; then
+    export _CA_CERTIFICATES_LOCKED="1"
```

Comment:
Alternative to `flock`, but cleanup can fail:

```
    (set -o noclobber; while ! > /run/ucc.lock; do sleep 0.5; done) 2>/dev/null
    trap 'rm /run/ucc.lock' EXIT
```

Reply:
too ugly for my taste

Reply:
busybox has flock too

Reply:
another alternative would be to call flock only if available. old behavior if not present which won't happen on a full system with util-linux

Reply:
Would work too

Reply:
updated